.github/ci/release_binary.sh

```diff
@@ -47,8 +47,8 @@ output=$(curl -fsS \
   -H "Authorization: token $GITHUB_TOKEN" \
   https://api.github.com/repos/$REPO/releases/generate-notes \
   -d '{"tag_name":"'$RELEASE_NAME'","previous_tag_name":"'$PREVIOUS_RELEASE_NAME'"}')
-# Code newlines as literal \n so thar curl understands that.
-BODY="$(jq -r '.body'   <<< $output | awk '{printf "%s\\n", $0}')"
+# Code newlines as literal \n and escape double quotes to make curl happy.
+BODY="$(jq -r '.body'   <<< $output | awk '{printf "%s\\n", $0}' | sed 's/"/\\"/g')"
```

Comment:
I'd argue "to generate valid JSON when the commit message uses quotes" is more accurate, I think curl will happily send our invalid JSON to the GitHub API which will then return 400 Bad Request.

Reply:
Good point - done. Thanks for the review Rodrigo!